spec/spec_helper.rb

```diff
@@ -50,8 +50,6 @@
     metadata[:type] ||= :model
   end
 
-  config.include Spec::Support::AuthHelper, :type => :view
-  config.include Spec::Support::ViewHelper, :type => :view
   config.include UiConstants,    :type => :view
```

Comment:
Note I'm removing UiConstants separately as there are some things dependent still within the project.